lib/db.js

```diff
@@ -54,11 +54,20 @@ class LimitDBRedis extends EventEmitter {
       redisOptions
     };
 
+    this.pingConfig = {
+      enabled: config.ping?.enabled || false,
```

Comment:
Since we are using a feature flag to control this behavior, this must be evaluated at all time. I would change this into a function () => Bool

Reply:
If `this.pingConfig` is not set, can we assume this is disabled?